src/sage/matrix/strassen.pyx

```diff
@@ -555,7 +559,7 @@ class int_range:
         if indices is None:
             self._intervals = []
             return
-        elif range is not None:
+        if range is not None:
```

Comment:
variable `range` will have to be renamed at some point.

Reply:
I have left this one as it is, because it appears as a named function argument.

Reply:
Indeed, it would need a `@rename_keyword` warning.